### ISSUE TITLE
[bitnami/clickhouse] toleration value doesn't work

### ISSUE THREAD

**Issue by zhenzh** [bitnami/charts#12861]
### Name and Version

bitnami/clickouse 0.2.1

### What steps will reproduce the bug?

install the helm chart with toleration provided

### Are you using any custom parameters or values?

values.yaml:
```
tolerations = [{
    key                     = "group"
    operator                = "Equal"
    value                   = "common"
    effect                  = "NoSchedule"
  }]
```

### What is the expected behavior?

install successfully

### What do you see instead?

install failed with error message that toleration configuration incorrect

### Additional information

after I updated statefulset.yaml line:
from
`tolerations: {{- include "common.tplvalues.render" (dict "value" $.Values.tolerations "context" .) | nindent 8 }}`
to
`tolerations: {{- include "common.tplvalues.render" (dict "value" $.Values.tolerations "context" $) | nindent 8 }}`
The installation with tolerations succeeded

**Comment by zhenzh** [comment]
I notice it has been fixed in the latest version 0.23.0 and it works well in my environment.

**Comment by javsalgar** [comment]
Awesome! Thanks for letting us know!
